gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -92,7 +99,14 @@ def get_variant_qc_annotation_resources(
     )
     run_vep = PipelineStepResourceCollection(
         "--run-vep",
-        output_resources={"vep_ht": get_vep(test=test)},
+        output_resources={
+            "vep_ht": get_vep(test=True)
+        },  # TODO: I found "test" is a boolean here.
```

Comment:
This needs to go back to:
```suggestion
        output_resources={"vep_ht": get_vep(test=test)},
```

Reply:
It's changed in my repo, I don't know why it's not shown on webpage. Can you see the correct one?